### PR TITLE
main Css work

### PR DESCRIPTION
**Pull request by ckdbswo**

### CODE REVIEW

File: .idea/dapp-rewrite.iml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (aman-mahalgavan) [inline .idea/dapp-rewrite.iml]: unnecessary file. Please list all such folders in .gitignore file

File: .idea/encodings.xml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (aman-mahalgavan) [inline .idea/encodings.xml]: unnecessary file. Please list all such folders in .gitignore file

File: .idea/inspectionProfiles/Project_Default.xml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<component name="InspectionProjectProfileManager">
```
Comment (aman-mahalgavan) [inline .idea/inspectionProfiles/Project_Default.xml]: unnecessary file. Please list all such folders in .gitignore file

File: .idea/misc.xml [hunk 4]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (aman-mahalgavan) [inline .idea/misc.xml]: unnecessary file. Please list all such folders in .gitignore file

File: .idea/modules.xml [hunk 5]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (aman-mahalgavan) [inline .idea/modules.xml]: unnecessary file. Please list all such folders in .gitignore file

File: .idea/vcs.xml [hunk 6]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (aman-mahalgavan) [inline .idea/vcs.xml]: unnecessary file. Please list all such folders in .gitignore file

File: .idea/workspace.xml [hunk 7]
```diff
@@ -0,0 +1,491 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (aman-mahalgavan) [inline .idea/workspace.xml]: unnecessary file. Please list all such folders in .gitignore file

File: .idea/inspectionProfiles/Project_Default.xml (hunk 3 above)
Comment (ckdbswo) [reply .idea/inspectionProfiles/Project_Default.xml]: I added .idea to the .gitignore file

File: .idea/misc.xml (hunk 4 above)
Comment (ckdbswo) [reply .idea/misc.xml]: I added .idea to the .gitignore file

File: .idea/modules.xml (hunk 5 above)
Comment (ckdbswo) [reply .idea/modules.xml]: I added .idea to the .gitignore file

File: .idea/vcs.xml (hunk 6 above)
Comment (ckdbswo) [reply .idea/vcs.xml]: I added .idea to the .gitignore file

File: .idea/workspace.xml (hunk 7 above)
Comment (ckdbswo) [reply .idea/workspace.xml]: I added .idea to the .gitignore file

**Pull request merged**